# Object description method
uint8 detection_method

# Supported types for object detection are: 
uint8 ARTAG = 1     # Requires the "tag_id" field to be filled
#uint8 SEGMENT = 2   # Requires the "primitive" field to be filled
#uint8 TEMPLATE = 3  # Requires the "mesh" field to be filled

# RMP
temoto_2/RMPRequest rmp

---

# Topics where raw tracking data is published. By default the data
# is published on one topic but there could be exceptions.
diagnostic_msgs/KeyValue[] output_topics

# RMP
temoto_2/RMPResponse rmp
